client/src/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -28,6 +31,27 @@ interface Props {
 
 export default function Login({ handleSubmit }: Props): JSX.Element {
   const classes = useStyles();
+  const { updateLoginContext } = useAuth();
+  const { updateSnackBarMessage } = useSnackBar();
+
+  //Demo User Function
+  const demoUser = () => {
+    const email = document.getElementById('email') as HTMLInputElement;
```

Comment:
Consider using refs to access DOM elements instead

Ref: https://www.newline.co/@dmitryrogozhny/how-to-access-dom-nodes-in-react-with-refs--57d97b3d